<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>M12U</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="serveurPort">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>50885</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="serveurIP">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIpServeur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelportServeur">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="listeMessages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pseudo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPseudo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>264</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pseudo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="message">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btQuitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonEnvoyer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonConnexion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
  </widget>
  <action name="actionConnexion">
   <property name="text">
    <string>Connexion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btQuitter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
